tests/integration/server/meeting/meeting.controller.test.ts

```diff
@@ -0,0 +1,1239 @@
+              beforeEach(async function () {
+                ([
+                  ...otherMeetings
+                ] = testCourseInstance.meetings);
+                newMeeting = new Meeting();
+                newMeeting.day = DAY.FRI;
+                newMeeting.startTime = '10:00:00-05';
+                newMeeting.endTime = '16:00:00-05';
+                const unbookedRoomQuery = roomRepository
+                  .createQueryBuilder('r')
+                  .where('"meetingTitle" IS NULL')
+                  .leftJoin(RoomBookingInfoView, 'rb', `rb.roomId = r.id AND rb.day = '${DAY.FRI}'`);
+                unbookedRoom = await unbookedRoomQuery
+                  .getOne();
+                newMeeting.room = unbookedRoom;
+                response = await request(api)
+                  .put(`/api/meetings/${testCourseInstance.id}`)
+                  .send({
+                    meetings: [
+                      ...otherMeetings,
+                      newMeeting,
+                    ].map(({ room, ...mtg }) => ({
+                      ...mtg, roomId: room?.id,
+                    })),
+                  });
+                result = response.body;
+                const otherIds = otherMeetings.map(({ id }) => id);
+                savedMeeting = result
+                  .find(({ id }) => (!otherIds.includes(id)));
+              });
+              it('Should include the new meeting in the result', function () {
+                strictEqual(savedMeeting.day, newMeeting.day);
+                const newStartTime = tzStringToAMPM(
+                  newMeeting.startTime
+                );
+                strictEqual(savedMeeting.startTime, newStartTime);
+                const newEndTime = tzStringToAMPM(
+                  newMeeting.endTime
+                );
+                strictEqual(savedMeeting.endTime, newEndTime);
+                strictEqual(savedMeeting.room.id, newMeeting.room.id);
+              });
+              it('Should add the new meeting in the database', async function () {
+                const dbMeeting = await meetingRepository.findOne(
+                  savedMeeting.id,
+                  { relations: ['room'] }
+                );
+                strictEqual(dbMeeting.day, newMeeting.day);
+                strictEqual(dbMeeting.startTime, newMeeting.startTime);
+                strictEqual(dbMeeting.endTime, newMeeting.endTime);
+                strictEqual(dbMeeting.room.id, newMeeting.room.id);
+              });
+            });
+            context('When the room is already booked', function () {
+              let bookedRoom: Room;
+              beforeEach(async function () {
+                ([
+                  ...otherMeetings
+                ] = testCourseInstance.meetings);
+                newMeeting = new Meeting();
+                newMeeting.day = DAY.FRI;
+                newMeeting.startTime = '10:00:00-05';
+                newMeeting.endTime = '16:00:00-05';
```

Comment:
I thought we were removing the timezone offset?

Reply:
We are, but that's a separate ticket and I didn't want to add even more changes onto this one PR. That does mean I'm going to have to change all of these when I do that ticket, but at least for now everything is working.

Reply:
Fair enough.